DeepSpeech.py

```diff
@@ -471,6 +471,11 @@ def train():
     best_dev_path = os.path.join(FLAGS.checkpoint_dir, 'best_dev')
     best_dev_filename = 'best_dev_checkpoint'
 
+    # Save flags next to checkpoints
+    flags_file = os.path.join(FLAGS.checkpoint_dir, 'flags.txt')
```

Comment:
nit: Might be a bit more complicated, but can't we have a placeholder of type `tf.string` that holds those values?

Reply:
You mean saving them in the checkpoint itself? The flags are an input for the construction of the graph, without which you can't restore the checkpoint, so that'd cause a bit of a chicken and egg problem.

Reply:
Right.